health-services/project-factory/src/server/utils/onGoingCampaignUpdateUtils.ts

```
@@ -0,0 +1,808 @@
+    else {
+      return null
+    }
+  } catch (error: any) {
+    throwError("PROJECT", 500, "PROJECT_FACILTY_SEARCH_ERROR")
+  }
+}
+
+
+async function fetchProjectStaffWithProjectId(request: any, projectId: any, staffId: any) {
+  try {
+    const { tenantId } = request?.body?.parentCampaign || request?.parentCampaign;
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)